src/main/java/views/PokemonFormViewController.java

```diff
@@ -89,33 +83,36 @@ private void notifyDataChangeListeners() {
     }
   }
 
-  private void getFormData() {
+  private boolean getFormData() {
```

Comment:
Segundo o guia de código limpo as funções devem ter apenas um ponto de retorno, então acredito que deveria ter uma variável `isValid = true;` no início do método e onde acontecer um erro marcar ela como `false` e no final do método dar um `return isValid`